src/index.js

```diff
@@ -53,7 +54,13 @@ function prettify(text, formatOptions) {
 
 function eslintFix(text, eslintConfig) {
   const eslintOptions = {
+    // overrideables
+    useEslintrc: false,
```

Comment:
This can be overridden. I don't think that it's advisable to do though or whether this would even be something you'd need to do. But I thought it'd be good to expose a way to override it.